evap/results/templates/result_widget_tooltip.html

```diff
@@ -1,5 +1,10 @@
 {% load evaluation_filters %}
-
+{% if show_weight_information and evaluation.course.evaluation_weight_sum and evaluation.course.evaluation_count > 1 %}
```

Comment:
In `results_index_evaluation.html`, you include `result_widget.html` and pass `show_weight_information=True`. However, in `result_widget.html`, where `result_widget_tooltip.html` is included, the variable is not explicitly passed, but only captured implicitly. (In the end, reading this if statement made me think: Huh, where did `show_weight_information` come from?)

I guess we'd want to make it explicit?